advancedbrowser/advancedbrowser/core.py

```diff
@@ -376,9 +399,25 @@ def closeEvent(self, evt):
         # Let Anki do its stuff now
         super(AdvancedBrowser, self).closeEvent(evt)
 
+    def negateUniqueNote(self):
```

Comment:
I prefer the word "toggle" instead of "negate", but it also makes it more consistent with other Anki stuff.